digos-ambassador/Content/Transformations/Species/snake/Leg.yml

```diff
@@ -0,0 +1,9 @@
+part: Leg
+species: snake
+description: "Transforms the target's {@side} leg into that of an anthro snake."
+default_base_colour: "dull grey"
+is_nsfw: false
+shift_message: "Smooth scales smother over {@target}'s {@side} leg. {@f|Their} muscles ripple and coil under {@f|their} skin, filling with potent and explosive muscle. Bones audibly crunch and pop as the arch of their foot stretches and their leg shifts into a predatory digitigrade stance, toes merging and melting into three large claw-tipped digits."
+grow_message: "The {@side} side of {@target}'s pelvis ripples and bulges, erupting into a digitigrade three-toed leg covered in smooth {@colour} scaled skin that sports rippling coils of muscle."
```

Comment:
I've noticed that almost every part mentions the muscles. All of the parts will be listed together in one strung-together description, and it might get repetitive. Cut down on the muscle descriptions to critical parts, or shorten them.

Reply:
I noticed this as well. It was an unfortunate side-effect of trying to describe the totality of the shift. Will crop.